src/panels.cpp

```diff
@@ -1826,9 +1827,32 @@ const std::string panel_manager::get_current_layout_id() const
     return current_layout_id;
 }
 
+int panel_manager::get_width_right()
+{
+    if( get_option<std::string>( "SIDEBAR_POSITION" ) == "left" ) {
+        return width_left;
+    }
+    return width_right;
+}
+
+int panel_manager::get_width_left()
+{
+    if( get_option<std::string>( "SIDEBAR_POSITION" ) == "left" ) {
+        return width_right;
+    }
+    return width_left;
+}
+
 void panel_manager::init()
 {
     load();
+    update_offsets( get_current_layout().begin()->get_width() );
+}
+
+void panel_manager::update_offsets( int x )
+{
+    width_right = x;
+    width_left = 0;
```

Comment:
looks like width left is always 0? If it works like this, why are we even storing this? Or is this a typo?

Reply:
That's to allow for panels to decide to use some space on the right *and* space on the left if need be. It isn't currently used but my understanding is panels will soon have the ability to not be restricted to one side of the screen.

That's why there's two values, and why `get_width_right()` and `get_width_left()` are mirrors at present. It's to ensure that `game.cpp` doesn't have to know where the sidebar is, only which parts of the screen are available for its use after the reduction of space for the sidebar(s).

Reply:
Cool, that makes sense to me